src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java

```diff
@@ -40,6 +41,9 @@
   public static void enterBigQueryProperties(String tableProp) throws InterruptedException, IOException {
     CdfStudioLocators.bigQueryProperties.click();
     CdfBigQueryPropertiesLocators.bigQueryReferenceName.sendKeys(AUTOMATION_TEST);
+    for (int i=0;i<30;i++) {
```

Comment:
Rather than using for loop to clear the value of the element. Can we use something like `CdfBigQueryPropertiesLocators.projectID.clear()`? Do we face any issues with `clear()`? Is there any better way to do it @rmstar, @tivv ?

Reply:
I found this workaround and looks much better, we can test this!
`CdfBigQueryPropertiesLocators.projectID.sendKeys(Keys.chord(Keys.CONTROL,"a", Keys.DELETE));`

Reply:
Yes, pressing backspace 30 times is hacky. @itsankit-google your solution looks like a better option if clear() doesn't work.

Reply:
btw, if clear() doesn't work it would be good to document the reason in a comment for future reference.

Reply:
Ankit suggested this link for the solution: https://github.com/cdapio/cdap-e2e-tests/blob/effa23140514bfbba8cb66e13098d1f0ecf4450d/src/main/java/io/cdap/e2e/utils/SeleniumHelper.java#L126

Reply:
SeleniumHelper has the link documented. Modified it to while loop as suggested 
 public static void replaceElementValue(WebElement element, String value) {
    while (!element.getAttribute("value").equals("")) {
      element.sendKeys(Keys.BACK_SPACE);
    }
    element.sendKeys(value);
  }